drv/stm32xx-sys-api/src/h7.rs

```diff
@@ -158,12 +151,9 @@ pub enum Peripheral {
     Swp = periph(Group::Apb1H, 2),
     Crsen = periph(Group::Apb1H, 1),
 
-    #[cfg(feature = "h7b3")]
-    Dfsdm1 = periph(Group::Apb2, 30), // B3 differs from 43/47
-
     Hrtim = periph(Group::Apb2, 29), // 43/47 only
```

Comment:
I'm confused by the comments in this file – there are many lines which was `43/47 only` but aren't gated on `h743`.

(we could also probably remove the references to the `B3` and `47` more generally, since they're both unsupported)

Reply:
The lines aren't consistently conditional because I felt the danger of having someone accidentally name a peripheral that isn't implemented on their chip was small. I would like to at least keep the comments; we could comment out the lines that are only on chips we don't support, if you'd prefer.